src/Tizen.MachineLearning.Service/Tizen.MachineLearning.Service/Service.cs

```diff
@@ -0,0 +1,696 @@
+        /// <since_tizen> 12 </since_tizen>
+        public void Start()
+        {
+            MLServiceError ret = Interop.Service.Start(_handle);
+            MLService.CheckException(ret, "Failed to start the service because of internal error");
+        }
+
+        /// <summary>
+        /// Stops the process of service instance.
+        /// </summary>
+        /// <feature>http://tizen.org/feature/machine_learning.service</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed to stop the service.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public void Stop()
+        {
+            MLServiceError ret = Interop.Service.Stop(_handle);
+            MLService.CheckException(ret, "Failed to stop the service because of internal error");
+        }
+
+        /// <summary>
+        /// Gets the TensorsInfo of the service instance.
+        /// </summary>
+        /// <param name="IsInput">The boolean value for choosing input or output to get.</param>
+        /// <param name="name">The name of node in the pipeline. You can set NULL if serivce is constructed from model configuration.</param>
+        /// <feature>http://tizen.org/feature/machine_learning.service</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public TensorsInfo GetTensorsInformation(bool IsInput, string name = "")
+        {
+            MLServiceError ret = MLServiceError.InvalidParameter;
+            IntPtr nameHandle = Interop.Util.StringToIntPtr(name);
+            IntPtr inHandle;
+
+            if (IsInput)
+                ret = Interop.Service.GetInputInformation(_handle, nameHandle, out inHandle);
+            else
+                ret = Interop.Service.GetOutputInformation(_handle, nameHandle, out inHandle);
+
+            MLService.CheckException(ret, "Failed to get TensorsInfo handle");
+
+            return TensorsInfo.ConvertTensorsInfoFromHandle(inHandle);
+        }
+
+        /// <summary>
+        /// Sets the information of the service instance.
+        /// </summary>
+        /// <param name="name">The name to be set.</param>
+        /// <param name="value">The value to be set.</param>
+        /// <feature>http://tizen.org/feature/machine_learning.service</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
```

Comment:
should add invalid-param exception.

Reply:
I think `ArgumentException` is invalid-param exception.
There was no InvalidParameter exception in the other examples either.